postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -40,11 +40,15 @@ instances:
 # Custom metrics
 # Below are some examples of commonly used metrics, which are implemented as custom metrics.
 # Uncomment them if you want to use them as is, or use as an example for creating your own custom metrics.
-# The format for describing custome metrics is identical with the one used for common metrics in postgres.py
+# The format for describing custom metrics is identical with the one used for common metrics in postgres.py
 # Be extra careful with ensuring proper custom metrics description format. If your custom metric does not work
 # after an agent restart, look for errors in the output of "agent status" command, as well as
 # /var/log/datadog/collector.log file.
 #
+# Please visit our help article for a guide on collecting custom metrics from PostgreSQL:
+# https://help.datadoghq.com/hc/en-us/articles/208385813-Postgres-custom-metric-collection-explained
```

Comment:
Why pointing users to a KB article showing the deprecated approach?

Reply:
I think switching the order of the old/new options so the new "supported" approach is on top may also help.

Reply:
That's a good point! I've re-arranged the options.